src/incentive-details.tsx

```diff
@@ -103,13 +103,16 @@ function formatStartDate(start_date: number, type: IncentiveType) {
   }
 }
 
+const absoluteRewiringURL = (path: string) =>
+  new URL(path, 'https://www.rewiringamerica.org').toString();
+
 const renderDetailRow = (key: number, incentive: IIncentiveRecord) => (
   <tr key={key} className={incentive.eligible ? '' : 'row--dimmed'}>
     <td>
       <a
         className="more-info-link"
         target="_blank"
-        href="https://www.rewiringamerica.org/${incentive.more_info_url}"
```

Comment:
I think this was right it just should have been tagged template literal not double-quotes? But the fix looks fine to me as well.

Reply:
This resulted in a double slash (because `more_info_url` has a leading slash) which works fine, but I figured might as well construct the URL the robust way.